awx/ui_next/src/screens/Inventory/shared/InventoryGroupsDeleteModal.jsx

```diff
@@ -18,12 +18,8 @@ const ListItem = styled.li`
   color: var(--pf-global--danger-color--100);
 `;
 
-const InventoryGroupsDeleteModal = ({
-  onAfterDelete,
-  isDisabled,
-  groups,
-  i18n,
-}) => {
+const InventoryGroupsDeleteModal = ({ onAfterDelete, isDisabled, groups }) => {
+  const { i18n } = useLingui();
```

Comment:
are you using this for anything?

Reply:
we are, but it seems that we don't need to anymore.  I'll keep it in to follow our pattern for now and we will write an issue to update/remove the unnecessary i18n in the future.